pkgs/development/python-modules/pytorch/default.nix

```diff
@@ -117,7 +117,7 @@ let
 in buildPythonPackage rec {
   pname = "pytorch";
   # Don't forget to update pytorch-bin to the same version.
```

Comment:
should we bump?

Reply:
Yeah, but I usually do separate PRs for them.

Reply:
See #127305 and #127296.